Makefile

```diff
@@ -8,12 +8,14 @@ oc_build_head:
 	oc start-build systems-clustering --from-archive build/HEAD.tar.gz --follow
 
 oc_cluster_train:
-	oc new-app --template systems-clustering-job \
-		--param AIOPS_TRAINING_DATE=${AIOPS_TRAINING_DATE} \
-		--param CEPH_KEY=${CEPH_KEY} \
-		--param CEPH_SECRET=${CEPH_SECRET} \
-		--param CEPH_ENDPOINT=${CEPH_ENDPOINT} \
-		--param CEPH_BUCKET=${CEPH_BUCKET}
+	oc new-app mlflow-experiment-job --param APP_IMAGE_URI=your-application-image-name\
```

Comment:
I think training and running the experiment are two different things.

The training would create a new model and store it in ceph.

The experiment works either on previous models, or creates a new one and reports some metrics. 
I think the current metric code only compares two clusters, which is ok for now.

So, I'd create a new task: `oc_run_experiment` and keep the `oc_cluster_train` task as is

Reply:
Sure, that makes sense. I have now created a separate task for this.